build.xml

```diff
@@ -73,7 +73,7 @@ To get started using Eclipse, execute "./build.py build-dev" and import the top-
         </then></if>
         <installIvy/>
         <ivy:buildlist reference="all.buildpath" settingsRef="ivy.toplevel">
-            <fileset dir="${omero.home}/components" includes="*/build.xml" excludes="**/tools/**, **/tests/**, **/bioformats/**"/>
+            <fileset dir="${omero.home}/components" includes="*/build.xml" excludes="**/tools/**, **/tests/**"/>
```

Comment:
Interestingly, as reported by @pwalczysko, it was still apparrently possible to build a server with a very old/outdated local `components/bioformats` folder.

With this commit, the build started failing while retrieving these dependencies. The fact these dependencies were classified under the `bf-deps` organization matches the old organization name used for the build when Bio-Formats was embedded as a submodule:

```
    ==== ome-artifactory: tried
      http://artifacts.openmicroscopy.org/artifactory/maven/bf-deps/objenesis/2.1/objenesis-2.1.pom
      -- artifact bf-deps#objenesis;2.1!objenesis.jar:
 ```

The solution was to properly clean up the Git repository, i.e. `rm -rf components/bioformats` so that the Bio-Formats artifacts with the correct organization could be picked up. /cc @mtbc